src/test/java/edu/harvard/iq/dataverse/api/DatasetsIT.java

```diff
@@ -2583,5 +2583,48 @@ public void testFilesUnchangedAfterDatasetMetadataUpdate() throws IOException {
                 .body("data.latestVersion.files[0].directoryLabel", equalTo("code"));
         
     }
+
+    @Test
+    public void testAddFileToDatasetTabIngest() throws IOException, InterruptedException {
```

Comment:
It would make sense to rename this test method, to make it clear that it is specifically testing *skipping* ingest; to differentiate it from all the other ingest tests in FilesIT etc.
(Thinking about it, maybe this method needs to be moved to FilesIT.java too?)

Reply:
I renamed it to testAddFileToDatasetSkipTabIngest and moved it to FilesIT.